deeplabcut/live/dlc_live.py

```diff
@@ -1,18 +1,16 @@
-"""
-DeepLabCut2.0 Toolbox (deeplabcut.org)
```

Comment:
why is the header dropped?

Reply:
my bad, my linter removing double docstrings (had it on auto b/c i had a problem w/ double generation at some point)

Reply:
I added them back in in an additional commit